comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -258,31 +250,59 @@ func (ia *inventoryagent) initData() {
 	ia.refreshMetadata()
 }
 
-func (ia *inventoryagent) fetchSecurityAgentMetadata() {
-	type configGetter interface {
-		GetBool(string) bool
-	}
+type configGetter interface {
+	GetBool(string) bool
+	GetString(string) string
+}
 
-	securityCfg := configGetter(ia.conf)
-	// We query the configuration from the security agent itself to have accurate data. If the security-agent isn't
+// getCorrectConfig tries to fetch the configuration from another process. When successful it returns a new
+// configuration object and true. If not, the local config and false is returned.
```

Comment:
Could you update this comment, which has been out of date for a bit? This method only returns the `configGetter`, there is no bool that can be used to tell if the config is remote vs local.